gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1694,6 +1794,30 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
         g_value_set_string (value, "");
       }
       break;
+    case PROP_INPUTRANKS:
+      if (prop->input_meta.num_tensors > 0) {
+        gchar *rank_str;
+
+        rank_str = gst_tensors_get_rank_string (prop, TRUE);
+
+        g_value_set_string (value, rank_str);
+        g_free (rank_str);
+      } else {
+        g_value_set_string (value, "");
+      }
+      break;
+    case PROP_OUTPUTRANKS:
+      if (prop->output_meta.num_tensors > 0) {
+        gchar *rank_str;
+
+        rank_str = gst_tensors_get_rank_string (prop, FALSE);
+
+        g_value_set_string (value, rank_str);
+        g_free (rank_str);
+      } else {
+        g_value_set_string (value, "");
+      }
+      break;
```

Comment:
Try to "Fix rank" at fixate.

Reply:
Both `inputranks` and `outputranks` are READONLY properties and they are only used in get_property() callback.
Shoud I fix it?